### ISSUE TITLE
Option to not output the 3D Object

### ISSUE THREAD

**Issue by gituser123456789000** [YvanYin/Metric3D#65]
The 3D Object take up a lot of space and may not be useful for many people.. If we had an option to skip the creation of this model, that would be good.

I also noticed that the 3D Object models were all upside-down. Maybe that's just me, but that's what I experienced.

**Comment by JUGGHM** [comment]
You are right. Generating pointclouds is time-and-space consuming. As a temporary patch, you could comment (add a # at the left of each line) for the following codes. (from Line313 to 321) https://github.com/YvanYin/Metric3D/blob/f8101cae8973086f4f54e2c1c9e7968c3a172444/mono/utils/do_test.py#L313-L321

**Comment by gituser123456789000** [comment]
Thank you very much for the follow-up
